src/EcommerceStatsBundle/Bridge/Chameleon/Twig/FormatterTwigExtension.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace ChameleonSystem\EcommerceStatsBundle\Bridge\Chameleon\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class FormatterTwigExtension extends AbstractExtension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return [
+            new TwigFilter('format_number', [$this, 'formatNumber'], ['is_safe' => ['html']]),
```

Comment:
Sounds like something that should be more general (CoreBundle, ViewRendererBundle)?

But on the other hand could be already present under this name in some projects...

Reply:
True. From what I have read, the best practice here seems to be to prefix bundle specific filters explicitly making this `chameleon_system_ecommerce_stats.format_number` (not sure if the period is supported).

Reply:
:heavy_check_mark: periods are not supported, filter is now named `chameleon_system_ecommerce_stats_format_number`